runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java

```diff
@@ -145,19 +157,19 @@ public void channelRead(ChannelHandlerContext ctx, Object m)
      * Channel event that is triggered when a new connected channel is created.
      *
      * @param ctx channel handler context
-     * @throws Exception
      */
     @Override
-    public void channelActive(ChannelHandlerContext ctx)
-        throws Exception {
-        log.info("channelActive: Outgoing connection established to: {} from id={}", ctx.channel().remoteAddress(), ctx.channel().localAddress());
+    public void channelActive(ChannelHandlerContext ctx) {
+        log.info("channelActive: Outgoing connection established to: {} from id={}",
+                ctx.channel().remoteAddress(), ctx.channel().localAddress());
+
+        // Note: Some fields in the header are unused during the handshake process.
+        HeaderMsg header = getHeaderMsg(0, CorfuMessage.PriorityLevel.NORMAL, 0,
+                UUID.fromString("00000000-0000-0000-0000-000000000000"), this.clientId, false, true);
```

Comment:
Let's make "00000000-0000-0000-0000-000000000000" a constant, put it in to a utility class and use the constant instead of a string

Reply:
Done.